packages/clarity-js/src/data/encode.ts

```diff
@@ -63,10 +63,12 @@ export default function(event: Event): void {
             queue(tokens);
             break;
         case Event.Upload:
-            tokens.push(track.sequence);
-            tokens.push(track.attempts);
-            tokens.push(track.status);
-            queue(tokens, false);
+            if (track) {
```

Comment:
when will track be null? I only see from one place we call encode(Event.Upload)

track = { sequence, attempts: transitData.attempts, status: xhr.status };
            // Send back an event only if we were not successful in our first attempt
            if (transitData.attempts > 1) { encode(Event.Upload); }

based on that, track will never be null. Just making sure that we are cognizant of adding value when we add checks and not bloat codebase.